.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- 
+Filling out this template is required. Do not delete the template when submitting a Pull Request!
+-->
+## PR Checklist
```

Comment:
Personally I'm not a fan of this kind of cumbersome checklist. Could we simplify it to just 3 or four header questions?

Reply:
OK, I've simplified it a good deal. What do you think now?